File: python/ley de ohm.py
def ley_de_ohm():
    print("Calculadora de la Ley de Ohm")
    print("Seleccione la variable que desea calcular:")
    print("1. Tensión (V)")
    print("2. Corriente (I)")
    print("3. Resistencia (R)")

    opcion = input("Ingrese el número de la opción (1/2/3): ")

    if opcion == '1':
        # Calcular Tensión (V)
        I = float(input("Ingrese la corriente (I) en amperios: "))
        R = float(input("Ingrese la resistencia (R) en ohmios: "))
        V = I * R
        print(f"La tensión (V) es: {V} voltios")

    elif opcion == '2':
        # Calcular Corriente (I)
        V = float(input("Ingrese la tensión (V) en voltios: "))
        R = float(input("Ingrese la resistencia (R) en ohmios: "))
        I = V / R
        print(f"La corriente (I) es: {I} amperios")

    elif opcion == '3':
        # Calcular Resistencia (R)
        V = float(input("Ingrese la tensión (V) en voltios: "))
        I = float(input("Ingrese la corriente (I) en amperios: "))
        R = V / I
        print(f"La resistencia (R) es: {R} ohmios")

    else:
        print("Opción no válida. Por favor, seleccione 1, 2 o 3.")

# Llamar a la función
ley_de_ohm()

File: javaScript/ley de ohm.js
function leyDeOhm() {
    console.log("Calculadora de la Ley de Ohm");
    console.log("Seleccione la variable que desea calcular:");
    console.log("1. Tensión (V)");
    console.log("2. Corriente (I)");
    console.log("3. Resistencia (R)");

    let opcion = prompt("Ingrese el número de la opción (1/2/3):");

    if (opcion === '1') {
        // Calcular Tensión (V)
        let I = parseFloat(prompt("Ingrese la corriente (I) en amperios:"));
        let R = parseFloat(prompt("Ingrese la resistencia (R) en ohmios:"));
        let V = I * R;
        console.log(`La tensión (V) es: ${V} voltios`);

    } else if (opcion === '2') {
        // Calcular Corriente (I)
        let V = parseFloat(prompt("Ingrese la tensión (V) en voltios:"));
        let R = parseFloat(prompt("Ingrese la resistencia (R) en ohmios:"));
        let I = V / R;
        console.log(`La corriente (I) es: ${I} amperios`);

    } else if (opcion === '3') {
        // Calcular Resistencia (R)
        let V = parseFloat(prompt("Ingrese la tensión (V) en voltios:"));
        let I = parseFloat(prompt("Ingrese la corriente (I) en amperios:"));
        let R = V / I;
        console.log(`La resistencia (R) es: ${R} ohmios`);

    } else {
        console.log("Opción no válida. Por favor, seleccione 1, 2 o 3.");
    }
}

// Llamar a la función
leyDeOhm();